### DISCUSSION TITLE
Define a Variable for ID of user running the workflow

### DISCUSSION THREAD

**Discussion by sabreensalama** [argoproj/argo-workflows#12269] [Ideas]
Hello,
I have a lot of developers and testers and they are running the workflows, I think it is good to know who started the workflow, to see this into the logs and the details of the run too.

**Comment by agilgur5** [comment]
If they are submitting via the UI and CLI, there is an existing [Workflow Creator](https://argo-workflows.readthedocs.io/en/latest/workflow-creator/) feature that will add `workflows.argoproj.io/creator` labels based on available information.
